rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        sum(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"})
```

Comment:
I don't see any time series with `namespace="gitops-service-argocd"` in AppSRE Grafana. Which apps are we expecting to be on that namespace?